ansible/roles/jenkins-worker/templates/systemd.service.j2

```diff
@@ -12,6 +12,7 @@ User={{ server_user }}
 Restart=always
 RestartSec=30
 StartLimitInterval=0
+WorkingDirectory=/home/iojs
```

Comment:
This shouldn't cause problems on other OSes and versions right?

Reply:
I don't believe so, we use the same username and home directory on all .. having said that, we use `{{ server_user }}` everywhere else so I'm going to change it now